src/chain/chain.rs

```diff
@@ -712,7 +654,7 @@ impl Chain {
             | NetworkEvent::Offline(_)
             | NetworkEvent::ExpectCandidate(_)
             | NetworkEvent::PurgeCandidate(_) => {
-                self.state() == &ChainState::Normal && self.our_info().is_quorum(proofs)
```

Comment:
Not sure we needed ChainState -> SectionChange as part of this change.
Not 100% sure it helps clarity, in particular, the Section does change when adding or removing a member, but it will have SectionChange::None. (Also make the change bigger than strictly needed).

Reply:
The motivation for the rename was to avoid the clash/confusion between `SharedState` and `ChainState` (and the corresponding variable names). I find `SectionChange` a decent enough name as it more explicitly describes what's going on ("state" is too generic). I'm always open to better suggestion though.

Regarding adding/removing elders: `SectionChange::None` means there is no ongoing change in progress. Adding/removing elders are instantaneous changes which don't involve any ongoing process.

Reply:
It is not so immediate though, when a candidate is voted Online, then it is added to Parsec, and the new_info, then the section info is voted, and then it is consensused. But if that helped for now, I'm ok with the rename, can be fixed latter if it is becoming an issue.

Reply:
I find the name `SectionChange` a bit vague too. maybe even `SectionStatus` and change None to Normal or `PrefixChange` where even None could make sense. the point with the ChainState::Normal was to indicate normal activity which ofc goes into a stalled state when we're changing prefixes in Split/Merge state. SectionChange::None in that convention sounds like we've stopped mutating this section. minor point anyways tho as long as the meaning behind it isnt lost.